src/quarkbaellchen.md

```diff
@@ -0,0 +1,32 @@
+# Quarkbällchen – Fried curd balls
+
+![An image of my finished curd balls](pix/quarkbaellchen.webp)
+
+“Quark” is a type of fresh dairy product. It's usually translated as curd cheese or cottage cheese.
+
+## Ingredients
+
+- 420ml quark (plain yoghurt can be used as a replacement)
+- 2 eggs
+- 6/7 spoons of sugar
+- 2 packets of vanilla sugar
+- 400g flour (500g if using yoghurt)
+- 2 packets of baking powder
+- 1l sunflower oil for frying
```

Comment:
```suggestion
- 1L sunflower oil for frying
```

`l` on it's own kind of looks like a 1, so it is usually capitalized in this case.